src/IECorePython/LRUCacheBinding.cpp

```diff
@@ -280,4 +353,7 @@ void IECorePython::bindLRUCache()
 		)
 	);
 
+	def( "testSerialLRUCacheRecursion", testSerialLRUCacheRecursion );
```

Comment:
I guess we were already doing this, but why are we binding tests in this way, rather than putting all that code in `test/IECore/LRUCacheThreadingTest.h/cpp` ?